### DISCUSSION TITLE
Recreate Graphs as from Example from SQL database

### DISCUSSION THREAD

**Discussion by xavierallem** [pyg-team/pytorch_geometric#9283] [Q&A]
I have a SQL database of my own where I have data relating to people meeting and the infection, I want to create a Infection dataset just like the example, Here are my questions
1) Is there a Infectious dataset example for reference in example? if not then how? since it has temporal data
2) How can I create the dataset just like Infectious dataset?
3) Where Can I learn Information about how exactly the nodes, graph is created in the dataset?

**Comment by rusty1s** [comment]
The `InfectionDataset` in PyG is a synthetic dataset to use to evaluate XAI methods, I am not sure if it fits your needs. How you want to create your infection dataset depends a bit on your needs and the use-case you have in mind. I assume you want to treat this as a binary classification task?

**Reply by xavierallem** [reply]
Hi, 
I wanted I wanted to test the infectious_ct1 dataset from [Link](https://chrsmrrs.github.io/datasets/docs/datasets/), Unfortunately I thought this was already implemented in pyg dataset as both were termed infectious. Is there an example similar to the infection_ct1? I wanted to create a neural network with similar dataset of infectious_ct1.

**Reply by rusty1s** [reply]
I am not sure if this works, but you may wanna try
```python
from torch_geometric.datasets import TUDataset

dataset = TUDataset(root, name="infectious_ct1")
```
Does that work?

**Reply by xavierallem** [reply]
![image](https://github.com/pyg-team/pytorch_geometric/assets/60181637/8a1736e6-4358-4760-a646-fbd00b2e01be)

```python
current_dir = os.getcwd()
path = osp.join(current_dir, '../Data/INFECTIOUS')
dataset = TUDataset(path, name='infectious_ct1',use_node_attr=True,use_edge_attr=True).shuffle()
```

using this for python notebook. Got the same error

**Reply by rusty1s** [reply]
Progress towards this will be tracked here: https://github.com/pyg-team/pytorch_geometric/issues/9351.
